README.md

```diff
@@ -1,6 +1,6 @@
 # @loopback/example-shopping
 
-[![Travis Build Status](https://travis-ci.com/strongloop/loopback4-example-shopping.svg?branch=master)](https://travis-ci.com/strongloop/loopback4-example-shopping)
+![Continuous Integration](https://github.com/strongloop/loopback4-example-shopping/workflows/Continous%20Integration/badge.svg)
```

Comment:
Looking at https://raw.githubusercontent.com/strongloop/loopback-next/master/README.md as an example, would it be:

```
[![Continuous Integration Status](https://github.com/strongloop/loopback4-example-shopping/workflows/Continous%20Integration/badge.svg)](https://github.com/strongloop/loopback4-example-shopping/actions?query=workflow%3A%22Continous+Integration%22)
```